### PR TITLE
Ensure ip_forward when deploying with th role. Closes #159

### PR DESCRIPTION
**Pull request by ulvida**
Accordig to [the issue #159](https://github.com/Stouts/Stouts.openvpn/issues/159) at the upstream role, I propose this patch.

--- 
Conforme a [la issue #159](https://github.com/Stouts/Stouts.openvpn/issues/159) del rol upstream, propongo este patch.

### CODE REVIEW

File: tasks/system/forwarding.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     sysctl_set: true
     state: present
     reload: true
-  when: not lookup('env', 'IN_MOLECULE') | d(true, true) | bool
+  when:  not ( lookup('env', 'IN_MOLECULE') ) | d(true, true) ) | bool
```
Comment (santiagomr) [inline tasks/system/forwarding.yml]: Did you get to try this? It gives me an error:

` FAILED! => {"msg": "The conditional check 'not ( lookup('env', 'IN_MOLECULE') ) | d(true, true) ) | bool' failed. The error was: template error while templating string: unexpected ')'.`

There is a parenthesis to spare

Comment (santiagomr) [reply tasks/system/forwarding.yml]: I suppose it was a typing error, and the intention was to leave it as below (line 20)

**Review by santiagomr — COMMENTED** [review]
I'm testing this, but the conditional still returns true and the task is skipped

File: tasks/system/forwarding.yml (hunk 1 above)
Comment (santiagomr) [reply tasks/system/forwarding.yml]: After reaching the same conclusion, I see that you had already solved it and commented [here](https://github.com/Stouts/Stouts.openvpn/commit/fcbeab7f3e88a8077cbacdf67b8977c87d85f6c7#r39188246)

The correct way for it to work in both scenarios (normal and molecule test) is:
```yaml
when: not lookup('env', 'IN_MOLECULE') | d(false, true) | bool
```

Comment (ulvida) [reply tasks/system/forwarding.yml]: Sorry, It was a failed commit. Fix hereafter. Maybe we can stash commits for merge.

**Comment by ulvida** [comment]
Let's abandon the present PR and work on PR  #6.

**Pull request closed without merging**